portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java

```diff
@@ -94,6 +128,21 @@ public void testCreateWithInvalidPayload_v3() {
     }
   }
 
+  @Test
+  public void testCreateWithInvalidPayload_v4() {
```

Comment:
naming tests like this makes them hard to read. Instead add a descriptive test name instead. testCreateCommentWithInvalidPayloadNonExistentKey

Same comment for others above as well.